### ISSUE TITLE
= is treating the right side selector as *

### ISSUE THREAD

**Issue by Mouvedia** [axr/core#171]
```
<root>
<a></a>
<b></b>
<c></c>
</root>
```

```
//this (currently) matches c as well (and shouldn't)
root b = a { background: #0 }
```

**Comment by jakepetroules** [comment]
Do we NEED this for 0.4.8?

**Comment by veosotano** [comment]
This has been fixed for 0.4.9 already (on the development branch).

**Comment by Mouvedia** [comment]
It was needed for the screencast.

**Comment by veosotano** [comment]
Fixed by 32e194ad388af979187fc7d9d0700dedfb308c6a
